ARTCC/High/CDR1 N864 TLA-ASNUD.txt

```diff
@@ -0,0 +1,19 @@
+;N864-X SCO 124.500
```

Comment:
Upon further research, I'm not sure this is a CDR1. The AIP states:

"Between TLA and ASNUD, established from 0001 on Saturday to 2359 on Sunday as an alternative route when AWY P600, to the south of Perth (PTH), has been made available to Portmoak for gliding activity. However, aircraft may flight plan to use this route at any time during the above hours. At certain times, notably when the airspace is required for military use, this weekend route will not be available. In these circumstances, all of AWY P600 will be retained for use."

The phrase _as an alternative route_ is what stands out to me. It's as if this doesn't imply that the airspace actually changes, just that when there is gliding activity at the weekends, this is the other (actually more direct) option. I'm going to bring this up somewhere else for discussion...

Reply:
I propose "Conditional - N864 (TLA-ASNUD)" @luke11brown

Reply:
Fixed in 0c17e24363549d3ea774f6d9b6bbe2b04a693413